### PR TITLE
Add generic script to rattle inputs

### PR DESCRIPTION
**Pull request by sidgoyal78**
[Don't merge]

### CODE REVIEW

File: rattle_run.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    np.random.seed(sys_id) # seed the sampler 
+    n = len(full_traj)
+    val = int(n * 0.2)
+    assert val > 0
+
+    # sample 20% of frames in the trajectory
+    frame_ids = np.random.choice(n, val, replace=False)
+
+    # set number of outer loop iterations to 1 
+    VASP_FLAGS['nsw'] = 1
```
Comment (mshuaibii) [inline rattle_run.py:38]: NSW=0 for singlepoints (What I've used for the dataset I've generated thus far)

@ktran9891 confirm?

**Comment by mshuaibii** [comment]
Looks good, NSW=0 only change needed.

File: rattle_run.py (hunk 1 above)
Comment (ktran9891) [reply rattle_run.py:38]: Sorry for super-late response. As per [VASP documentation](https://www.vasp.at/wiki/index.php/NSW), the behavior of `NSW` changes depending on the value of `IBRION`. I'm not entirely sure what value of `IBRION` we used in this context.

I assume everything still worked out?

File: rattle_run.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    for frame in frame_ids:
+        image = full_traj[frame]
+        rattled_image = image.copy()
+        rattled_image.rattle(stdev=std, seed=100) # fixed seed for all inputs
```
Comment (iorin0217) [inline rattle_run.py:43]: Hi!
Fixing the seed of rattle() displaces all inputs in the same directions.
Is this your intended phenomenon?

Comment (sidgoyal78) [reply rattle_run.py:43]: We fixed the seed as:
- the input catalyst+adsorbate systems were not the all same, and,
- even for a given system, we sampled a few frames from a trajectory, and all frames had different positions of atoms.
This made sure that we got diverse samples.

cc @mshuaibii

**Pull request closed without merging**